src/main/java/uk/gov/justice/hmpps/casenotes/legacy/service/CaseNoteService.java

```diff
@@ -321,4 +327,19 @@ private boolean isAllowedToCreateRestrictedCaseNote() {
     private boolean isAllowedToViewOrCreateSensitiveCaseNote() {
         return securityUserContext.isOverrideRole("POM", "VIEW_SENSITIVE_CASE_NOTES", "ADD_SENSITIVE_CASE_NOTES");
     }
+
+    private PersonCaseNoteEvent generateEvent(PersonCaseNoteEvent.Type type, OffenderCaseNote caseNote) {
+     return new PersonCaseNoteEvent(
+         type,
+         caseNote.getPersonIdentifier(),
+         caseNote.getId(),
+         caseNote.getLegacyId(),
+         caseNote.getSubType().getType().getCode(),
+         caseNote.getSubType().getCode(),
+         CaseNoteRequestContext.get().getSource(),
```

Comment:
This must always be DPS I believe because it's hard coded in CaseNoteContextInterceptor